source/Game.py

```diff
@@ -38,5 +42,20 @@ def one_step(self):
     def set_active_tetramino(self, tetramino_key):
         self.active_tetramino = Tetramino(tetramino_key)
 
-    def print_spawn_matrix(self):
-        size = len(self.active_tetramino.tetramino_matrix)
+    def print_spawned_tetramino(self):
```

Comment:
Strange stuff occuring here -- 
self.PLAY_AREA and self.PLAY_AREA[0] should both have been type `list` but the print statement on the next line shows that the 1st one is `list` while the 2nd is a `string`